[2023-11-09T23:08:10.870+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: early_sepsis_prediction.preprocess_mean_input_norm manual__2023-11-09T23:08:03.529267+00:00 [queued]>
[2023-11-09T23:08:10.885+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: early_sepsis_prediction.preprocess_mean_input_norm manual__2023-11-09T23:08:03.529267+00:00 [queued]>
[2023-11-09T23:08:10.886+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-11-09T23:08:10.887+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-11-09T23:08:10.888+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-11-09T23:08:10.903+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): preprocess_mean_input_norm> on 2023-11-09 23:08:03.529267+00:00
[2023-11-09T23:08:10.913+0000] {standard_task_runner.py:55} INFO - Started process 6299 to run task
[2023-11-09T23:08:10.921+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'early_sepsis_prediction', 'preprocess_mean_input_norm', 'manual__2023-11-09T23:08:03.529267+00:00', '--job-id', '273', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmpwucxrihh']
[2023-11-09T23:08:10.929+0000] {standard_task_runner.py:83} INFO - Job 273: Subtask preprocess_mean_input_norm
[2023-11-09T23:08:11.028+0000] {task_command.py:388} INFO - Running <TaskInstance: early_sepsis_prediction.preprocess_mean_input_norm manual__2023-11-09T23:08:03.529267+00:00 [running]> on host 911414a99fc3
[2023-11-09T23:08:11.123+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Pavs
AIRFLOW_CTX_DAG_ID=early_sepsis_prediction
AIRFLOW_CTX_TASK_ID=preprocess_mean_input_norm
AIRFLOW_CTX_EXECUTION_DATE=2023-11-09T23:08:03.529267+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-09T23:08:03.529267+00:00
[2023-11-09T23:08:11.208+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/base.py:493: FutureWarning: The feature names should match those that were passed during fit. Starting version 1.2, an error will be raised.
Feature names unseen at fit time:
- Bilirubin_direct
- EtCO2
- Fibrinogen
- TroponinI
Feature names seen at fit time, yet now missing:
- BaseExcess
- Glucose
- HR
- Hct
- Platelets

  warnings.warn(message, FutureWarning)

[2023-11-09T23:08:11.211+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/src/pylab1.py", line 153, in preprocess_mean_input_norm
    test_df.iloc[:,1:-1] = scaler.transform(test_df.iloc[:,1:-1])
  File "/home/airflow/.local/lib/python3.7/site-packages/sklearn/preprocessing/_data.py", line 501, in transform
    reset=False,
  File "/home/airflow/.local/lib/python3.7/site-packages/sklearn/base.py", line 585, in _validate_data
    self._check_n_features(X, reset=reset)
  File "/home/airflow/.local/lib/python3.7/site-packages/sklearn/base.py", line 401, in _check_n_features
    f"X has {n_features} features, but {self.__class__.__name__} "
ValueError: X has 35 features, but MinMaxScaler is expecting 36 features as input.
[2023-11-09T23:08:11.241+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=early_sepsis_prediction, task_id=preprocess_mean_input_norm, execution_date=20231109T230803, start_date=20231109T230810, end_date=20231109T230811
[2023-11-09T23:08:11.259+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 273 for task preprocess_mean_input_norm (X has 35 features, but MinMaxScaler is expecting 36 features as input.; 6299)
[2023-11-09T23:08:11.291+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-11-09T23:08:11.315+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
